src/pages/EnablePayments/AdditionalDetailsStep.js

```diff
@@ -114,7 +115,7 @@ class AdditionalDetailsStep extends React.Component {
      * @returns {Object}
      */
     validate(values) {
-        const errors = {};
+        let errors = {};
```

Comment:
Can be const after applying the requested changes below
```suggestion
        const errors = {};
```